spec/factories/trips.rb

```diff
@@ -0,0 +1,8 @@
+FactoryBot.define do
+  factory :trip do
+    lat { "59.334" }
+    lng { "18.063" }
+    name { "Stockholm" }
```

Comment:
So, is it `name` or is it `destination` Is your factory test passing?

Reply:
There wasn't one at first. It does now however.